lib/cortex/snippets/client.rb

```diff
@@ -21,6 +21,16 @@ def snippet(request, options = {}, block)
         end
       end
 
+      def snippet_content(request, options = {}, content)
```

Comment:
This seems to share a lot of logic with the `snippet` method - do you think you could abstract the common logic to a private method, or use this method in the `snippet` method, to DRY things up a bit?